pkg/card/cards.go

```diff
@@ -1,8 +1,8 @@
 package card
 
 import (
-	"example/paws-quest/pkg/models"
 	"example/paws-quest/pkg/action"
```

Comment:
do you have a plugin or smth to reorg imports ?

Reply:
yes, I use https://github.com/daixiang0/gci